GiniVision/Classes/Core/Extensions/Collection.swift

```diff
@@ -8,10 +8,27 @@
 
 import Foundation
 
+extension Collection {
+    var isNotEmpty: Bool {
+        return !isEmpty
+    }
+}
+
 internal extension Collection where Iterator.Element == CFString {
     
     var strings: [ String ] {
         return self.map { $0 as String }
     }
     
 }
+
+extension Collection where Iterator.Element == GiniVisionDocument {
+    var isAssorted: Bool {        
```

Comment:
I just feel that Assorted doesn't translate well into English. At least I don't associate it with a mixture of different typed things.

I'd use something simple like:
* `containsDifferentTypes`
* `mixedDocumentTypes`

Idk, I also cannot find a well-suited name

Reply:
Yeah, maybe make sense to be more explicit in this case since this extension might be public in a future PR. I will go with the `containsDifferentTypes `